docs/tutorials/data-modeling-lifecycle/part-4-extending-solution-model.md

```diff
@@ -1,5 +1,168 @@
 ## Extending Solution Model
 
-!!! warning annotate "Warning"
+This tutorial demonstrates how to extend a solution model. Note that extending a model means changing it
+by adding, reshaping, or removing any of its elements **after it has been put in production**.
 
-    This tutorial is planned but not yet available. Please check back later.
+We assume that you already have a solution model built on top of an Enterprise Data Model. This
+tutorial will be an extension of the [Analytic Solution Model](./part-2-analytic-solution),
+with the Enterprise Data Model from [Extending an Enterprise Data Model](./part-3-extending-enterprise-model).
+
+## Introduction
+
+After getting the core concepts from his analytic solution model into the Enterprise Data Model, Olav
+wants to update his solution model to utilize the concepts from the Enterprise Data Model. This will enable
+him to write his timeseries forecast which can then be utilized by the trading department.
+
+## Downloading the Current Solution Model
+
+Olav starts by using **NEAT** to download the enterprise model. He opens **NEAT** and selects the `Import DMS`
+workflow, and then clicks on the `Import DMS` step. This opens the modal with the configuration for the import
+
+<img src="../../artifacts/figs/life_cycle_download_reference_model_analytic_soluteion_model.png" height="300">
+
+Olav selects the following options:
+
+* **Data model id**: This is the ID of the current analytic solution model.
+* **Reference data model id** This is the ID of the enterprise model that the solution model is based.
+* **Report formatter**: This is used in the validation of the model. The current solution model should be valid,
+  so this is likely not needed.
+* **Role**: This is which format Olav wants to download the model. He selects `dms_architect` as he wants the
+  model in the implementation format as he will focus on how data is stored in the model.
+
+Furthermore, he clicks on the `Create Excel Sheet` step which opens a modal with the configuration for the export
```

Comment:
Where is this step? I cannot find it

Reply:
I think you mean `RulesToExcel` ?

Reply:
No, it is `Create Excel File` in the `Import_DMS` workflow